js/components/DiskII.tsx

```diff
@@ -11,8 +11,8 @@ import { loadJSON, loadHttpFile, getHashParts } from './util/files';
 export interface DiskIIData {
     number: 1 | 2;
     on: boolean;
-    name?: string;
-    side?: string;
+    name?: string | undefined;
```

Comment:
To me, it seems like this should be non-optional.

Reply:
This is the drive itself, so when there is no disk, name could be undefined. However we do normally display a placeholder, so depending on where I'm setting that placeholder this could be made non-optional.